modules/electrophysiology_browser/help/view_session.md

```diff
@@ -0,0 +1,17 @@
+# View Session
+
+This page displays information from an individual recording session.
+
+Summary information about the candidate, the recording session and the hardware used is displayed.
+
+File packages can be downloaded containing just the recording signal, the events, or other metadata, or all files compressed together, for the session in view. These file packages are organized according to the EEG-BIDS standard<sup id="1">[1](#f1)</sup>, and are available for download as follows:
```

Comment:
```suggestion
Files can be downloaded containing just the recording signal, the events, or other metadata, or all files compressed together, for the current session. These files are organized according to the EEG-BIDS standard<sup id="1">[1](#f1)</sup>, and are available for download as follows:
```
* `current session` is clearer than `session in view`
* more accurate now as to what is a package and what is not